homeassistant/components/zwave_me/sensor.py

```diff
@@ -38,28 +64,22 @@
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    "meterElectric_power_factor": SensorEntityDescription(
+        key="meterElectric_power_factor",
+        device_class=SensorDeviceClass.POWER_FACTOR,
+        native_unit_of_measurement="PF",
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
     "meterElectric_voltage": SensorEntityDescription(
         key="meterElectric_voltage",
         device_class=SensorDeviceClass.VOLTAGE,
         native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
         state_class=SensorStateClass.MEASUREMENT,
     ),
-    "light": SensorEntityDescription(
```

Comment:
Why do you remove these three sensors?

Reply:
We don't have such a sensor on our side. Our developer accidentally copied it from HA types. See above the luminance

Reply:
We don't have them either - the HA integration developer is not skilled in Z-Way, so he focused on temperature and a few others he had in devices on the table. The rest was copy&paste. We are not fixing it after additional testing.